packages/govuk-frontend/src/govuk/components/checkboxes/_index.scss

```diff
@@ -29,39 +24,42 @@
   }
 
   .govuk-checkboxes__input {
-    $input-offset: ($govuk-touch-target-size - $govuk-checkboxes-size) / 2;
-
-    position: absolute;
-
-    z-index: 1;
-    top: $input-offset * -1;
-    left: $input-offset * -1;
-
     width: $govuk-touch-target-size;
     height: $govuk-touch-target-size;
     margin: 0;
-
     opacity: 0;
-
     cursor: pointer;
   }
 
   .govuk-checkboxes__label {
-    display: inline-block;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
```

Comment:
Have we got an example set up for this?

I couldn't replicate the alignment issues without it:

>* Use flexbox on labels so that their content is always centre aligned against the input